go.mod

```diff
@@ -53,6 +54,7 @@ require (
 	github.com/opencontainers/runtime-spec v1.0.3-0.20220825212826-86290f6a00fb
 	github.com/opencontainers/runtime-tools v0.9.1-0.20221014010322-58c91d646d86
 	github.com/opencontainers/selinux v1.10.2
+	github.com/pkg/errors v0.9.1
```

Comment:
nit: this is something that's obsoleted since Go 1.13 (when native %w-style error wrapping was introduced). If possible, please do not add it back.

Reply:
Argh, sorry and thanks for spotting that. I though I got rid of all of them but missed that one. Removed.